src/Common/TimeoutMap.ts

```diff
@@ -0,0 +1,20 @@
+export type TimeoutMap = {
```

Comment:
`OmnichannelEndpointsTimeoutConfig`? We have `OmnichannelEndpoints` which contains the list of endpoints